bigbluebutton-html5/imports/ui/components/tooltip/component.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Tippy from 'tippy.js';
+import _ from 'lodash';
+import cx from 'classnames';
+
+const propTypes = {
+
+};
+
+const defaultProps = {
+  position: 'bottom'
+};
+
+class Tooltip extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      tippySelector: _.uniqueId('tippy-'),
```

Comment:
Should`nt be a state since it never changes. Just use as a property of the object. `this.selectorId`